eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/io/DbsnpVariantsWriterTest.java

```diff
@@ -914,6 +954,374 @@ public void declusterRs638662487() throws Exception {
+                                                                                          submittedVariant4);
+        DbsnpClusteredVariantEntity clusteredVariantEntity3 = buildClusteredVariantEntity(clusteredVariantAccession3,
+                                                                                          clusteredVariant);
+        DbsnpClusteredVariantEntity clusteredVariantEntity4 = buildClusteredVariantEntity(clusteredVariantAccession3,
+                                                                                          clusteredVariant);
+
+        DbsnpVariantsWrapper wrapper3 = new DbsnpVariantsWrapper();
+        wrapper3.setClusteredVariant(clusteredVariantEntity3);
+        wrapper3.setSubmittedVariants(Collections.singletonList(submittedVariantEntity3));
+
+        DbsnpVariantsWrapper wrapper4 = new DbsnpVariantsWrapper();
+        wrapper4.setClusteredVariant(clusteredVariantEntity4);
+        wrapper4.setSubmittedVariants(Collections.singletonList(submittedVariantEntity4));
+
+        // when
+        dbsnpVariantsWriter.write(Arrays.asList(wrapper, wrapper2, wrapper3, wrapper4));
+
+        // then
+        assertClusteredVariantStored(1, wrapper);
+        DbsnpSubmittedVariantEntity expectedSubmittedVariantEntity2 = changeRS(submittedVariantEntity2,
+                                                                               clusteredVariantEntity1.getAccession());
+        DbsnpSubmittedVariantEntity expectedSubmittedVariantEntity3 = changeRS(submittedVariantEntity3,
+                                                                               clusteredVariantEntity1.getAccession());
+        assertSubmittedVariantsStored(2, submittedVariantEntity1, expectedSubmittedVariantEntity3);
+        assertSubmittedVariantsHaveActiveClusteredVariantsAccession(wrapper.getClusteredVariant().getAccession(),
+                                                                    expectedSubmittedVariantEntity2,
+                                                                    expectedSubmittedVariantEntity3);
+        assertSubmittedUpdateOperationsHaveClusteredVariantAccession(5, 1, clusteredVariantAccession2);
+        assertSubmittedUpdateOperationsHaveClusteredVariantAccession(5, 2, clusteredVariantAccession3);
+        assertSubmittedVariantMergeOperationStored(5, 1, submittedVariantEntity1);
+        assertSubmittedVariantMergeOperationStored(5, 1, submittedVariantEntity3);
+        // could be 3 operations total? 2 copies for clusteredVariantEntity3
+
+        assertEquals(0, mongoTemplate.count(new Query(), DBSNP_CLUSTERED_VARIANT_DECLUSTERED_COLLECTION_NAME));
+    }
+
+    /**
+     * Clustered variant rs136611820 was merged into several other clustered variants with the same hash: [42568024,
+     * 42568025]
+     * <p>
+     * This happened because rs136611820 had several locations. One matched the location of rs42568024 and other matched
```

Comment:
"had several locations" is misleading as we discussed offline. Re-phrase to "had locations that matched with that of multiple ss IDs".